Lecture2/Task3/com/playtika/NewStudent.java

```diff
@@ -0,0 +1,130 @@
+package com.playtika;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class NewStudent {
```

Comment:
What is a difference between `Student` and `NewStudent`?

Reply:
Different names to avoid  the conflicts.

Reply:
I assume we don't need 2 classes